### DISCUSSION TITLE
Welcome to xor-gate-perceptron Discussions!

### DISCUSSION THREAD

**Discussion by itsAnanth** [itsAnanth/xor-gate-perceptron#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Knightrider16** [comment]
What exactly is binary cross entropy loss doing here?

**Reply by itsAnanth** [reply]
Since we're doing a binary classification problem with XOR dataset, an apt loss function has to be used. generally BCE loss is prefered for binary classification as it nicely alligns with the output probability distribution during maximum likelihood estimation
